src/components/sections/About.section.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import AboutAirBalloon from '../../assets/about/AboutAirBalloon.svg';
+import AboutSection from '../../assets/about/AboutMeSection.svg';
+
+const About: React.FC = () => {
+    const [offsetY, setOffsetY] = useState(0);
+
+    const handleScroll = () => {
+        setOffsetY(window.scrollY * 0.45); // Handles the speed at which the balloon moves
+    };
+
+    useEffect(() => {
+        window.addEventListener('scroll', handleScroll);
+
+        return () => window.removeEventListener('scroll', handleScroll);
+    }, []);
+    return (
+        <div className="relative bg-gradient-to-b from-peachPuff to-brightUbe pb-10 pt-48">
+            {/* MAIN SVG (Background) */}
+            <div className="relative left-[-4%] w-[115%] overflow-hidden">
+                <img src={AboutSection} alt="Billboard" className="block" />
```

Comment:
And we don't actually need to provide alt text to the decorative images like top border, hawk svg...
If it is decorative, we could just remove them.

Link: [Creating an Empty or Null Alt Attribute for Decorative Images](https://pressbooks.ulib.csuohio.edu/accessibility/chapter/chapter-3-6-creating-an-empty-or-null-alt-attribute-for-decorative-images/#:~:text=Decorative%20images%20require%20the%20addition,between%20the%20set%20of%20quotes.)

A decorative image is one that doesn’t convey meaning nor add information to the page. Examples of decorative images include:

- Images that serve as borders, spacers, corners, flourishes
- Images adjacent to text that convey the same meaning or information as the text
- Images adjacent to text that serve as “eye-candy,” or “ambience,” and don’t add information.
- Images used next to link text simply to improve its appearance or increase its clickable area

Reply:
This is awesome, I did not know that. I fixed it for the Hawk Balloon. @teoh4770 Should I keep it for the main background though, since it has content? (i.e., the hackathon statistics)

Reply:
Apology for the late comment, if the image conveys message, then we should add alt text, that's what I thought.

Reply:
All good! Just wanted to make sure :) thanks!